scripts/drone/rgm.star

```diff
@@ -166,28 +166,21 @@ def rgm_copy(src, dst):
       dst: destination of the files.
 
     Returns:
-      Drone steps.
+      Drone step.
     """
     commands = [
         "printenv GCP_KEY_BASE64 | base64 -d > /tmp/key.json",
         "gcloud auth activate-service-account --key-file=/tmp/key.json",
         "gcloud storage cp -r {} {}".format(src, dst),
     ]
 
-    if not dst.startswith("gs://"):
```

Comment:
Hm... isn't that now done everywhere where you use that function or did I miss a place? :)

Reply:
it's not in used in the custom build function. In fact I think it'd break it if it did.

Reply:
Hm... my impression was that the upload URL would start with `gs://`, hence the question :)

Reply:
This function would read it as `$${UPLOAD_TO}` since it's an environment variable. If we wanted to test the contents of it, this condition would need to be written in bash in the `commands` list. :\